src/app/Command.h

```diff
@@ -75,7 +75,7 @@ class Command
      *  @retval #CHIP_NO_ERROR On success.
      *
      */
-    CHIP_ERROR Init(Messaging::ExchangeManager * apExchangeMgr, InteractionModelDelegate * apDelegate);
+    CHIP_ERROR Init(chip::Messaging::ExchangeManager * apExchangeMgr, InteractionModelDelegate * apDelegate);
```

Comment:
Having to prefix all Messaging with "chip::" is pretty annoying.

Would doing `chip::app::clusters::ClusterName` help resolve the ambiguities?  And then places that want to avoid writing out "clusters::" can (not in headers) use the right namespace.

Another option would be `chip::app::ClusterNameCluster`.....

Reply:
I changed the code to use `chip::app::Clusters`.